exampleCourse/questions/demoOverlayDropdown/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-overlay width="700" height="700" clip="false">
```

Comment:
I think it would be great to have the option to have the field blank at the start.

Reply:
That's a good idea. It will be done. Thanks!

Reply:
*Blank field and example added*

![image](https://user-images.githubusercontent.com/7872295/84928059-a513d980-b082-11ea-9a8d-d6c3291b9c56.png)

Reply:
@andrewstec how would you feel if we borrowed the styles from `pl-*-input` to emphasize content?

![Screen Shot 2020-06-17 at 4 39 00 PM](https://user-images.githubusercontent.com/833642/84953440-21291400-b0b9-11ea-9672-dfe726ac3c03.png) 

/ cc @nicknytko

Reply:
I would say that's terrific. I did not know about that. I love it.

Reply:
Yeah, agreed with @coatless in maybe stealing styles from the `pl-string-input` element.  I’d also suggest that when no answer is selected it generates an “invalid” submission to keep consistency with the other input fields.

Reply:
I will try to get this out of the way early in the morning. These seem like quick tasks. Thanks!

Reply:
The style change will need a bit of extra markup in mustache. I don't think there's a need to modify the CSS as the new styles use bootstrap styling. c.f. https://github.com/PrairieLearn/PrairieLearn/blob/134b9ce4ddff40ba82d09c3c054ee7fd7785ef2a/elements/pl-string-input/pl-string-input.mustache#L12-L44

Reply:
This is the new look: 
![image](https://user-images.githubusercontent.com/7872295/85042478-1e253680-b140-11ea-9174-7d36e7b7dd9a.png)

Reply:
![image](https://user-images.githubusercontent.com/7872295/85043422-2fbb0e00-b141-11ea-9d44-fa95dc505dd5.png)

Just testing this style. This might be too hard to read.

Reply:
This is also a bootstrap option that is used throughout PL (minus the colours): 
![image](https://user-images.githubusercontent.com/7872295/85043632-6c870500-b141-11ea-8c8a-91e8344ca193.png)

Reply:
Does anyone have a strong preference on the way forward here? If we want, we can go with your original idea, but play around with the padding or margins and font-size. Maybe it is best to just use the last option?

Reply:
I am sticking with this: 

![image](https://user-images.githubusercontent.com/7872295/85045140-6f82f500-b143-11ea-8595-328dcec1284b.png)

I will leave it this way, unless anyone requests that it is changed :)

Reply:
Done here: b38fe0daf9341ff7c4bd70078aeea71016a79822.